docs/UserGuide.md

```diff
@@ -130,85 +149,77 @@ Format: `list`
+ * Only the name or IC number is searched.
+ * Only full words will be matched e.g. `Han` will not match `Hans`
+ * For the name, only persons matching at least one keyword will be returned (i.e. `OR` search).
+   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 
 Formats:
-1. `find n/NAME` *or* `id/IC_NUMBER`
-2. `find n/NAME [field]` *or* `id/IC_NUMBER [field]`
-
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name or IC number is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* For the name, only persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
-
-Acceptable values for each parameter:
-* String
+ 1. `find n/NAME` *or* `id/IC_NUMBER`
+ 2. `find n/NAME [field]` *or* `id/IC_NUMBER [field]`
 
 Expected outputs when the command succeeds:
-* `Patient n/NAME or id/IC_NUMBER: [field] …`
+ * `Patient n/NAME or id/IC_NUMBER: [field] …`
 
 Expected output when the command fails:
-* `Unable to find the patient. Check if the patient’s information is correct.`
+ * `Unable to find the patient. Check if the patient’s information is correct.`
 
 Examples:
-* `find n/John` returns `john` and `John Doe`
-* `find id/s894219J phone/ address/` returns phone and address of patient with IC number s894219J
-* `find id/S872D` returns `Alex Yeoh`, with IC number `S872D` <br>
-  ![result for 'find id/S872D'](images/findidS872DResult.png)
+ * `find n/John` returns `john` and `John Doe`
+ * `find id/s894219J p/ a/` returns phone and address of patient with IC number s894219J
+ * `find id/S872D` returns `Alex Yeoh`, with IC number `S872D` <br>
+   ![result for 'find id/S872D'](images/findidS872DResult.png)
 
 ### Deleting a person or field: `delete`
 
 Deletes the specified person or the fields for the person from HealthSync
 
-Patient information includes:
-- Patient’s contacts
-- Patient’s medical history
-- Patient’s ward information
-- Patient’s upcoming appointment time(s)
-
 Format: `delete n/NAME or id/IC_NUMBER [field]`
 
-* Deletes the person with the specified `n/NAME or id/IC_NUMBER`.
-* The name or ic number must be a valid input
-* To delete a specified field only instead of the entire person, we indicate the field behind of the identification
-* If multiple people has the same name, HealthSync will display a list of people with that name together with their IC number.
+ * Deletes the person with the specified `n/NAME or id/IC_NUMBER`.
+ * The name or ic number must be a valid input
+ * To delete a specified field only instead of the entire person, we indicate the field behind of the identification
+ * If multiple people has the same name, HealthSync will display a list of people with that name together with their IC number.
 
 Acceptable values for each parameter:
-* String
+ * String
```

Comment:
Just a small issue here: will we have this "Acceptable values for each parameter" section for all commands? I noticed that this section was removed in the previous commit for the find command.

Reply:
Ooh, yea I must have missed the removal of the section header.

I think the "Acceptable values" section is probably unnecessary since we already have the common fields section above this. A section like this would probably also necessitate rather technical language for a UG. It is rather formally understood what format a phone number field or informal date field would take too, so I think it's probably unnecessary in a UG.